chrome_manifest_v3/views/class_popupview.js

```
@@ -1161,21 +1144,21 @@ class PopupView {
   }
 
   // Select/de-select attachments and images based on first button's state:
-  toggleCheckboxes(tag) {
+  toggleCheckboxes_deprecated(tag) {
     const $jTags = $('#' + tag + ' input[type="checkbox"]', this.$popup);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)